weblate/addons/migrations/0002_remove_addon_project_scope_addon_project_and_more.py

```diff
@@ -0,0 +1,48 @@
+# Copyright © Michal Čihař <michal@weblate.org>
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def populate_project_from_component(apps, schema_editor):
+    """Migrate Addons with project_scope from using component to using project."""
+    Addon = apps.get_model("addons", "Addon")
+    for addon in Addon.objects.filter(project_scope=True):
+        addon.project = addon.component.project
+        addon.component = None
+        addon.save(update_fields=["project", "component"])
```

Comment:
Use `bulk_update` for better performance:

```suggestion
    updates = []
    for addon in Addon.objects.filter(project_scope=True):
        addon.project = addon.component.project
        addon.component = None
        updates.append(addon)
        if len(updates) > 1000:
             Addon.objects.bulk_update(updates, ["project", "component"])
             updates = []
    if updates:
         Addon.objects.bulk_update(updates, ["project", "component"])
```